luigi_pipeline/lib/model/sv_mt_schema.py

```diff
@@ -0,0 +1,240 @@
+import hail as hl
+
+from lib.model.base_mt_schema import BaseMTSchema, row_annotation, RowAnnotationOmit
+from lib.model.seqr_mt_schema import SeqrGenotypesSchema, SeqrVariantsAndGenotypesSchema
+
+from hail_scripts.computed_fields import variant_id
+
+
+BOTHSIDES_SUPPORT = "BOTHSIDES_SUPPORT"
+PASS = "PASS"
+
+# Used to filter mt.info fields.
+CONSEQ_PREDICTED_PREFIX = 'PREDICTED_'
+NON_GENE_PREDICTIONS = {'PREDICTED_INTERGENIC', 'PREDICTED_NONCODING_BREAKPOINT', 'PREDICTED_NONCODING_SPAN'}
+
+
+INTERVAL_TYPE = 'array<struct{type: str, chrom: str, start: int32, end: int32}>'
+
+
+def get_cpx_interval(x):
+    # an example format of CPX_INTERVALS is "DUP_chr1:1499897-1499974"
+    type_chr = x.split('_chr')
+    chr_pos = type_chr[1].split(':')
+    pos = chr_pos[1].split('-')
+    return hl.struct(type=type_chr[0], chrom=chr_pos[0], start=hl.int32(pos[0]), end=hl.int32(pos[1]))
+
+
+class SeqrSVVariantSchema(BaseMTSchema):
```

Comment:
It seems like several of these are direct copy-pastes from `BaseSeqrSchema`. Might it be petter to have this inherit from that instead, or to move some of those annotations down form `BaseSeqrSchema` to `BaseMTSchema`?

Reply:
I think my thought there was that a couple fields were (`contig` and `start`) but that most were not, and that there were more things on `BaseSeqrSchema` that I did not want to inherit.  I'm ok with moving those few things to `BaseMTSchema` though!